docs/ibc/README.md

```diff
@@ -0,0 +1,5 @@
+If you're looking for an overview of the IBC protocol, read the documents in numerical order.
```

Comment:
Add `# Inter-Blockchain Communication Protocol (IBC)` before. 

+ maybe a short intro paragraph (like an abstract) before the following sentence "For an overview of the IBC protocol, read the following documents in numerical order.", as this will probably be the first point of contact with IBC

Reply:
```suggestion
For an overview of the IBC protocol, read the following documents in numerical order.
```

Reply:
So added, so updated.